reference-data/loan-policy-storage/loan-policies/rolling.json

```diff
@@ -6,8 +6,8 @@
   "loansPolicy" : {
     "profileId" : "Rolling",
     "period" : {
-      "duration" : 1,
-      "intervalId" : "Months"
+      "duration" : 60,
+      "intervalId" : "Days"
```

Comment:
What is the purpose for this change?

Reply:
Cate asked for it to be made when we talked about what policy would best help them test